dear_petition/petition/etl/load.py

```diff
@@ -31,14 +31,11 @@ def import_ciprs_records(files, user, parser_mode, batch_label=""):
     logger.info(f"Created batch {batch.id}")
     for idx, file_ in enumerate(files):
         logger.info(f"Importing file {file_}")
-        if settings.CIPRS_SAVE_PDF:
```

Comment:
Should we remove this setting if we're not using it here?

Reply:
Done!